packages/flutter_test/lib/src/controller.dart

```diff
@@ -1567,6 +1608,7 @@ abstract class WidgetController {
     double touchSlopX = kDragSlopDefault,
     double touchSlopY = kDragSlopDefault,
     PointerDeviceKind kind = PointerDeviceKind.touch,
+    int? viewId,
```

Comment:
Can this be a `FlutterView?` instead of an `id?`. It can be difficult to figure out where this `id` should come from, but you can find out how to acquire a `FlutterView` from its docs.